src/main/java/io/temporal/samples/helloworld/HelloWorldStarter.java

```diff
@@ -0,0 +1,31 @@
+package io.temporal.samples.helloworld;
+// @@@SNIPSTART java-hello-world-sample-workflow-starter
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class HelloWorldStarter {
```

Comment:
Naming nit: The notion of "starter" bothered me as I played with the samples. It executes the app logic that runs the workflow. It acts as a client or user of the workflow service. Yeah, "client" is already used as a proxy for connecting to the service. Would it be better to name HelloWorldStarter as HelloWorldUser or HelloWorldApp or HelloWorldCaller or something like that?

Reply:
What about "initiator"?